src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -11078,23 +11107,23 @@ definitions:
       description:
         description: Description of the rule
         type: string
-      enabled:
+      isEnabled:
         description: Whether the rule is enabled
         type: boolean
         default: true
-      cascade:
-        description: Whether the rule also applies to subfolders
+      isInheritable:
+        description: Whether the rule can also apply to subfolders
         type: boolean
         default: false
-      asynchronous:
+      isAsynchronous:
         description: Whether the rule should be run in the background
         type: boolean
         default: false
       errorScript:
         description: If the rule should be run in the background then an optional error script can be referenced
         type: string
-      shared:
-        description: Whether the rule has been shared with more than one folder
+      isShared:
```

Comment:
Am I missing something or was this not declared with the other fields above (line 11097)?

Reply:
Line 11097 is part of:
```
    required:
      - id
      - name
      - actions
      - isEnabled
      - isInheritable
      - isAsynchronous
```

It controls which fields are mandatory (in the response)

Reply:
Thanks for the clarification Tom